dts/arm/st/f0/stm32f091.dtsi

```diff
@@ -52,5 +52,21 @@
 				label = "GPIOE";
 			};
 		};
+
+		dma1: dma@40020000 {
+			interrupts = <9 0 10 0 10 0 11 0 11 0 11 0 11 0>;
+		};
+
+		dma2: dma@40020400 {
+			compatible = "st,stm32-dma";
+			#dma-cells = <4>;
+			reg = <0x40020400 0x400>;
+			interrupts = <9 0 10 0 10 0 11 0 11 0 11 0
+				      11 0 10 0 10 0 11 0 11 0 11 0>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x2>;
+			st,mem2mem;
+			status = "disabled";
+			label = "DMA_2";
+		};
 	};
 };
```

Comment:
Shouldn't this change (dma2) apply also to F051 and F071?

Reply:
But F051 and F071 has only one DMA instance according to the datasheet.

Reply:
Ok, correct.
I'm reading RM0091, and it seems infact that only STM32F091 has DMA2.
Sorry.